pdf_compression/README.rst

```diff
@@ -0,0 +1,115 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+PDF Compression & PDF/A
+=======================
+
+This addon provides the ability to convert outgoing and imported PDF attachments.
+You can compress the PDFs to save space in you database or file system.
+Also the possibility exists to convert PDFs in PDF/A. Currently implemented:
+
+* PDF/A-1b
+* PDF/A-2b (recommend to use this setting because it is faster and result in smaller files)
+
+
+Installation
+============
+
+#. You need at least Gostscript 9.18 (I recommend 9.19)
+   `http://ghostscript.com <http://ghostscript.com/download/gsdnld.html>`_
+
+
+Configuration
+=============
+
+If the odoo module is installed:
+
+#. You can define ``Compession level`` and ``PDF/A TYPE`` via
+   `Settings -> Configuration -> Knowledge`.
+#. Also you can override PDF/A option for individual Reports apart from Global Settings via
+   `Settings -> Technical -> Reports -> Reports -> Select a Report -> PDF/A archiving`.
+
+
+
+Usage
+=====
+
+
+#. Compression rate outcome:
```

Comment:
Explain here that you have to print a report that is saved as attachment, and the attachment will be compressed. I understand that the save prompt is for the non compressed PDF. Isn't it?